stage-3--rfc-template.md

```diff
@@ -10,9 +10,12 @@
   of the accepted proposal.
 -->
 
+**If you have feedback, please leave it on the Stage 3 PR (link below).**
```

Comment:
But is Stage 2 the right place for that once the feature is available as experimental? Genuinely curious

Reply:
A Stage 2 RFC should be closed at this point. It isn't specified in the `README.md`, and I'll update that. 

If a feature is available as experimental, Stage 3 is the right place for feedback.

Reply:
Alright this is exactly the kind of explanation I want as part of the clarification PR.

```suggestion
**If you have feedback and the feature is released as experimental, please leave it on the Stage 3 PR. Otherwise, comment on the Stage 2 issue (links below).**
```

wdyt?

Reply:
Looks good to me!